src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_buffer_fusing.cpp

```diff
@@ -208,21 +209,31 @@ bool concat_in_place_optimization::match(concatenation_node& node) {
         // Check that there isn't already some padding between inputs in concat axis.
         // If node has already been optimized we skip this check - this is just cascade adjustment.
         if (!node.can_be_optimized()) {
-            if (idx != node.get_dependencies().size() && input_padd.upper_size().raw[concat_axis] != 0)
+            if (idx != node.get_dependencies().size() && input_padd.upper_size().sizes(def_fmt)[concat_axis] != 0)
                 return false;
-            if (idx != 0 && input_padd.lower_size().raw[concat_axis] != 0)
+            if (idx != 0 && input_padd.lower_size().sizes(def_fmt)[concat_axis] != 0)
                 return false;
         }
 
-        lower_padd_in_axis += input->get_output_layout().size.raw[concat_axis];
+        lower_padd_in_axis += input->get_output_layout().size.sizes(def_fmt)[concat_axis];
         idx += 1;
     }
 
     return true;
 }
 
 void concat_in_place_optimization::optimize_cascade(concatenation_node& node, std::list<concatenation_node*>& need_reoptimization) {
+    auto def_fmt = format::get_default_format(node.get_output_layout().get_rank());
```

Comment:
Suppose it makes sence to add a variable like `out_layout` instead of `node.get_output_layout()` to simplify the code.

Reply:
fixed